CDCgov/prime-reportstream
# 7448 [Bug] Json Diff Editor gets out-of-sync when doing edits

Updated onChange method of JSON diff editor to reformat the text so that the changed text gets highlighted correctly

Test Steps:
1. Go into Edit Settings for `ignore` (`admin/orgsettings/org/ignore`)
2. Get into the Json Diff editor via the `Preview Save...` button
3. Change the Right Hand editable side by editing an existing field... but do NOT use correct json lint format. For example: change `"filters": [],` to `"filters": ["1","2"],` (which really should be:
```
"filters": [
    "1",
    "2"],
```
to be correctly formatted.
Result: The highlighting for the differences is now correct.
![Screenshot 2022-12-09 at 1 53 24 PM](https://user-images.githubusercontent.com/102491809/206801954-68e728a5-3c41-4c8d-8eb6-b1b6bc893d5f.png)

## Checklist

### Testing
